bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/presentation-download-dropdown/component.jsx

```diff
@@ -91,69 +91,73 @@ class PresentationDownloadDropdown extends PureComponent {
     };
 
     if (!isDownloadable) {
-      this.menuItems.push(
-        {
-          key: this.actionsKey[0],
-          dataTest: 'enableOriginalPresentationDownload',
-          label: intl.formatMessage(intlMessages.enableOriginalPresentationDownload),
-          onClick: () => toggleDownloadOriginalPresentation(true),
-        },
-      );
-    } else {
-      this.menuItems.push(
-        {
-          key: this.actionsKey[0],
-          dataTest: 'disableOriginalPresentationDownload',
-          label: intl.formatMessage(intlMessages.disableOriginalPresentationDownload),
-          onClick: () => toggleDownloadOriginalPresentation(false),
-        },
-      );
-    }
-
-    if (hasAnnotations) {
-      this.menuItems.push(
-        {
-          key: this.actionsKey[1],
-          id: 'sendAnnotatedDocument',
-          dataTest: 'sendAnnotatedDocument',
-          label: intl.formatMessage(intlMessages.sendAnnotatedDocument),
-          onClick: () => {
-            closeModal();
-            handleDownloadingOfPresentation('Annotated');
+      if (hasAnnotations) {
+        this.menuItems.push(
+          {
+            key: this.actionsKey[0],
+            dataTest: 'enableOriginalPresentationDownload',
+            label: intl.formatMessage(intlMessages.enableOriginalPresentationDownload),
+            onClick: () => toggleDownloadOriginalPresentation(true),
+          },
```

Comment:
Why is this part inside the `if (hasAnnotations) {` if it always happens?
Move it outside the condition and avoid duplicating code!